build.gradle

```diff
@@ -78,10 +78,6 @@ allprojects {
                         'PreferSafeLoggableExceptions',
                         'PreferSafeLogger'
 
-                // increase strictness for built-in error-prone checks
-                error((com.google.errorprone.scanner.BuiltInCheckerSuppliers.ENABLED_WARNINGS +
-                        com.google.errorprone.scanner.BuiltInCheckerSuppliers.DISABLED_CHECKS
-                        ).collect { it.canonicalName() } as String[])
```

Comment:
These imports were the main reason for needing `com.google.errorprone:error_prone_core` on gradle classpath. Going to switch over to not-yet-merged or released `AllSuggestionsAsWarnings` from https://github.com/google/error-prone/pull/3301 to opt-in to more strict checks